app/controllers/content_items_controller.rb

```diff
@@ -136,16 +136,41 @@ def set_guide_draft_access_token
     @content_item.draft_access_token = params[:token]
   end
 
+  class GdsApi::Response
+    def update_response_body
+      updated_response = JSON.parse(@http_response.body).dig("data", "edition").deep_transform_keys(&:underscore)
+      @http_response = RestClient::Response.create(
+        updated_response.to_json,
+        @http_response.net_http_res,
+        @http_response.request,
+      )
+    end
+  end
+
   def load_content_item
     content_item = Services.content_store.content_item(content_item_path)
 
-    content_item["links"]["ordered_related_items"] = ordered_related_items(content_item["links"]) if content_item["links"]
-
-    @content_item = PresenterBuilder.new(
-      content_item,
-      content_item_path,
-      view_context,
-    ).presenter
+    @content_item = if content_item["schema_name"] == "news_article" && (Features.graphql_feature_enabled? || params.include?(:graphql))
```

Comment:
**thought:** query params

If the query param was `graphql=false` 👀

Reply:
I was thinking about this same thing again the other day.

It was when I was debugging the broken Ministers Index page in integration. The feature flag being on currently means always use GraphQL. The feature flag is on in integration, so I'd've had to switch it off (a helm charts pull request) to test the page with Content Store instead.

Would changing this behaviour so that the feature flag just influenced the default make things too complicated?
* if there's no query param set, check the feature flag
* if query param is set, do what it says

Reply:
> * if there's no query param set, check the feature flag
> * if query param is set, do what it says

That seems like a good approach to me.